src/server/string_family.cc

```diff
@@ -1170,6 +1259,63 @@ auto StringFamily::OpMGet(bool fetch_mcflag, bool fetch_mcver, const Transaction
   return response;
 }
 
+void StringFamily::ClThrottle(CmdArgList args, ConnectionContext* cntx) {
```

Comment:
Since this algorithm is not well known, please add a few lines explaining the meaning of the arguments and 
how the command looks like. For example, how it is shown in redis code:
https://github.com/redis/redis/blob/unstable/src/t_list.c#L535